### ISSUE TITLE
Endless loading when username has blankspace

### ISSUE THREAD

**Issue by Mi3012** [owncloud/contacts#298]
I tried with various names, everytime I use a blankspace in the name (like Test User) the contacts app won't stop loading. I tried the fix to add 'enable_avatars' => false, in config.php but nothing happens. But names like Testuser work fine. 

(in owncloud 8.1 it worked)

**Comment by Henni** [comment]
I can't reproduce this.
@Mi3012 can you share a vcf file with us?

**Comment by Mi3012** [comment]
Sry I don't mean the name of an contact I mean as ownCloud username. I have users with dots like T.User which work with the app but the users with blankspace like Test User see a endless loading symbol when they open the app:

<img width="1194" alt="bildschirmfoto 2016-03-26 um 13 24 02" src="https://cloud.githubusercontent.com/assets/18085058/14060016/2ad6e9ac-f356-11e5-98cf-a73e1f398733.png">

**Comment by Henni** [comment]
@Mi3012 Thanks for the clarification

**Comment by skjnldsv** [comment]
This is what I got

```
dav.js?v=9eb9407…:798 
Uncaught (in promise) TypeError: Cannot read property 'props' of undefined(…)
    callee$0$0$             @ dav.js?v=9eb9407…:798
    tryCatch                @ dav.js?v=9eb9407…:121
    invoke                  @ dav.js?v=9eb9407…:294
    Gp.(anonymous function) @ dav.js?v=9eb9407…:330
    onFulfilled             @ dav.js?v=9eb9407…:3743

```

**Comment by DeepDiver1975** [comment]
The issue is here - https://github.com/owncloud/contacts/blob/master/js/dav/lib/accounts.js#L98

account.principalUrl is encoded but response.href is not and because of that the response is not found.

We'll address this with 1.3 - maybe already by having the new dav client lib in place - let's see

**Comment by irgendwie** [comment]
fixed by #502.
